LoopKitHostedTests/HKObserverQueryMock.swift

```diff
@@ -12,6 +12,14 @@ import HealthKit
 class HKObserverQueryMock: HKObserverQuery {
     let updateHandler: (HKObserverQuery, @escaping HKObserverQueryCompletionHandler, Error?) -> Void
     
+    @available(iOS 15.0, *)
```

Comment:
[nit] Not sure what this has to do with this PR. Consider creating a new "iOS 15 support" Jira ticket and moving this there.